MultisigTests/Logic/Models/Transaction/TransactionTests.swift

```diff
@@ -60,4 +60,56 @@ class TransactionTests: XCTestCase {
         }
         sema.wait()
     }
+
+    func testTransactoinEncodedData() throws {
+        let tx = Transaction(to: "0x8e6A5aDb2B88257A3DAc7A76A7B4EcaCdA090b66",
+                             value: "1000123",
+                             data: "0x561001057600080fd5b5060405161060a3803806106",
+                             operation: .delegate,
+                             safeTxGas: "21000",
+                             baseGas: "29943",
+                             gasPrice: "99123001",
+                             gasToken: "0x1001230000000000000000000000000000000001",
+                             refundReceiver: AddressString(.zero),
+                             nonce: "12305734256",
+                             safeTxHash: nil)
+
+        let domainHashInput = oneline("""
+035aff83d86937d35b32e04f0ddc6ff469290eef2f1b692d8a815c89404d4749
+000000000000000000000000092CC1854399ADc38Dad4f846E369C40D0a40307
+""")
+        let domainHash = EthHasher.hash(Data(ethHex: domainHashInput))
+
+        let valueHashInput = oneline("""
+bb8310d486368db6bd6f849402fdd73ad53d316b5a4b2644ad6efe0f941286d8
+0000000000000000000000008e6A5aDb2B88257A3DAc7A76A7B4EcaCdA090b66
+00000000000000000000000000000000000000000000000000000000000F42BB
+ef8553f949acc5f0cb8002523b7a4f8e02664b6637eddc74ad72bb8e38588309
+0000000000000000000000000000000000000000000000000000000000000001
+0000000000000000000000000000000000000000000000000000000000005208
+00000000000000000000000000000000000000000000000000000000000074F7
+0000000000000000000000000000000000000000000000000000000005E87F39
+0000000000000000000000001001230000000000000000000000000000000001
+0000000000000000000000000000000000000000000000000000000000000000
+00000000000000000000000000000000000000000000000000000002DD7A9A70
+""")
+        let valueHash = EthHasher.hash(Data(ethHex: valueHashInput))
+
+        let txHashInput = "1901" + domainHash.toHexString() + valueHash.toHexString()
```

Comment:
what is 1901? can we add a named constant?

Reply:
ok